AsyncDisplayKit/ASTextNode.mm

```diff
@@ -435,28 +425,20 @@ + (void)drawRect:(CGRect)bounds withParameters:(ASTextNodeDrawParameters *)param
       UIRectFillUsingBlendMode(CGContextGetClipBoundingBox(context), kCGBlendModeCopy);
     }
   }
-
+  
   // Draw shadow
-  [[parameters.renderer shadower] setShadowInContext:context];
```

Comment:
Although the object creation was the main cost, it's awesome to see this diff remove 10 or more method calls to the parameters object as well!